server/application-server/src/main/java/de/tum/in/www1/hephaestus/leaderboard/SlackMessageService.java

```diff
@@ -57,26 +63,47 @@ public void sendMessage(String channelId, List<LayoutBlock> blocks, String fallb
         }
     }
 
+    private List<LeaderboardEntryDTO> getTop3SlackReviewers() {
+        LocalDate after = LocalDate.now().minusDays(8);
+        LocalDate before = LocalDate.now().minusDays(1);
```

Comment:
Kinda depends on how we schedule the CRON and the timeframe we want.
The default CRON is set to execute at Tuesday 9am. This means that with `minusDays(1)` - `minusDays(8)` we get exactly Monday to Monday.
To be fair, since we include the last date as well, I should change it to Monday-Sunday to be in sync with the leaderboard.

Reply:
Wouldn't it make sense to post to post? We can also let the leaderboard go from post to post, i.e. Tuesday 9am to Tuesday 9am

Reply:
That would be possible as well of course. Although we would have to consider that we are currently only not persisting the time anywhere (LocalDate doesn't store it and neither does the state in the URL), so this would require quite a bit of refactoring.
Let's discuss this idea in tomorrow's meeting maybe?